project-file/Khadeeja-Gilani-SP18-BSE-114.md

```diff
@@ -0,0 +1,26 @@
+Project Name:
+  EmoBot (Mental Health Assistant)
+  
+Project Owner:
+  Khadeeja Gilani, Midhat Zahra 
+
+Roll Number:
+  SP18-BSE-114
+
+Project Details:
+Description:
+  EmoBot is an application created to provide users with a platform to share their mental health issues and get therapy and meditation recommendations according to that. User can have a conversation with the chatbot and share their concerns and get a solution after the bot predicts their mental health issue and mood etc. The user can also perform Cognitive Behavioural Therapies that the system provides. Furthermore, a user can track their mood history with daily and average statics visualized in the form of graphs. They can also view their health progress according to the therapy results and health tests. If there are users who have other severe mental health issues that our system cannot handle, the user is redirected to the locate therapist page to locate nearby therapists on google map to get professional help.
+
+Technology Stack:
+  Python is used for back-end programming for training the chatbot and other machine learning algorithms used in providing recommendations.
+  Flutter is used for user interface in front-end development.
+  Firebase Firestore is used as a data store for our application.
+
+Deployment or Distribution Strategy:
+  An android application is developed so it will be deployed on Google Play Store.
```

Comment:
@SyedaKhadeejaGilani How will you deploy it on Google Play Store, please write it down.